functions/src/events/Event.model.ts

```diff
@@ -0,0 +1,4 @@
+export type EventRequest = {
+  name: string;
+  [key: string]: any;
```

Comment:
I guess `any` is to keep it flexible with what we pass in? Should prob just add an `eslint-ignore` even though it's just a warning